source/boards/sharing-boards.rst

```diff
@@ -21,7 +21,7 @@ Sharing Boards
 To share a board with others:
 
 1. Open a board in any view.
-2. On the top-right of the board select the options menu **...**, then select **Share board**.
+2. On the top-right of the board select **Share board**.
 3. Set the toggle on to **Publish to web and share this board with anyone**.
```

Comment:
```suggestion
3. Copy the internal link to **Share an editable view with users who have permissions to the board**.
-- OR --
4. Switch to the Publish tab and set the toggle on to **Publish and share a "read-only" link with everyone on the web**.
```

Reply:
There's an extra "internal link" option in the Share modal. Maybe using numbered bullet points isn't the best with the "OR" option, but added details on the new functionality. Feel free to fix up the formatting as you see fit @justinegeffen.